ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionFragment.kt

```diff
@@ -155,7 +155,7 @@ class DataCollectionFragment : AbstractFragment(), BackPressListener {
   }
 
   private fun onTaskChanged(taskPosition: TaskPosition) {
-    viewPager.currentItem = taskPosition.absoluteIndex
+    viewPager.setCurrentItem(taskPosition.absoluteIndex, false)
```

Comment:
Please add named arg to clarify what `false` refers to

Reply:
added comment.
As this is a java function, it doesn't support named argument as of kotlin.